server/athenian/api/defer.py

```diff
@@ -81,17 +80,23 @@ def launch():
             transaction = Transaction(name="defer", sampled=False, hub=Hub.current)
         else:
             parent = Hub.current.scope.transaction
-            transaction = Transaction(name="defer " + parent.name,
-                                      sampled=parent.sampled,
-                                      op=parent.op,
-                                      description=parent.description,
-                                      hub=Hub.current)
+            if parent is None:
```

Comment:
Can you please change `if Hub.current.scope.span is None:` to `if (parent := Hub.current.scope.transaction) is None:` because it seems that the current check is wrong (IDK how because a span cannot live without a parent transaction).

Reply:
done